config.toml

```diff
@@ -0,0 +1,3 @@
+Username = "mrgo123"
```

Comment:
Let's actually leave these fields blank to make it really obvious that the user needs to fill them in.